samples/helm-deploy/templates/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "carbon-aware-sdk.fullname" . }}
+  namespace: {{ .Values.namespace }}
```

Comment:
We should use `.Release.namespace` rather than the value from values.yaml. https://helm.sh/docs/chart_template_guide/builtin_objects/